src/main/java/org/sonarlint/intellij/finding/issue/vulnerabilities/LocalTaintVulnerability.kt

```diff
@@ -51,6 +51,8 @@ class LocalTaintVulnerability(private val primaryLocationPsiFile: PsiFile?, priv
 
   override fun getCleanCodeAttribute(): CleanCodeAttribute? = remoteTaintVulnerability.cleanCodeAttribute.orElse(null)
   override fun getImpacts(): Map<SoftwareQuality, ImpactSeverity> = remoteTaintVulnerability.impacts
+  override fun getHighestQualityImpact(): Map.Entry<SoftwareQuality, ImpactSeverity>? =
+    getImpacts().entries.stream().max(java.util.Map.Entry.comparingByValue()).orElse(null);
```

Comment:
And here. Another random idea, instead of having a Map everywhere we could have a wrapper type